compiler/rustc_feature/src/active.rs

```diff
@@ -666,6 +666,10 @@ declare_features! (
     /// Allows qualified paths in struct expressions, struct patterns and tuple struct patterns.
     (active, more_qualified_paths, "1.54.0", Some(80080), None),
 
+    /// Allows creation of instances of a struct by moving fields that have
+    /// not changed from prior instances of the same struct (RFC #2528)
+    (active, type_changing_struct_update, "1.55.0", Some(86618), None)
```

Comment:
This just needs a trailing comma.

Reply:
Also, the version number will need to be updated, right?

Reply:
Yes, I believe it should be updated to 1.57.0 (because 1.56.0 is about to be promoted to beta).